### PR TITLE
style(all): changes to comply with lint

### PR DESCRIPTION
**Pull request by z3dev**
These changes are syntax only, changes to correct lint warnings.

The Github CI has also been bumped to use Node 16 and Node 18, which is still behind the LTS versions.

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
* [x] Does your submission pass tests?

### CODE REVIEW

**Comment by z3dev** [comment]
@platypii @hrgdavor please review

File: packages/modeling/src/maths/OrthoNormalBasis.js [hunk 1]
```diff
@@ -106,7 +106,7 @@ OrthoNormalBasis.GetCartesian = function (xaxisid, yaxisid) {
   } else {
     throw new Error('OrthoNormalBasis.GetCartesian: invalid combination of axis identifiers. Should pass two string arguments from [X,Y,Z,-X,-Y,-Z], being two different axes.')
   }
-  return new OrthoNormalBasis(new Plane(new Vector3D(planenormal), 0), new Vector3D(rightvector))
+  // return new OrthoNormalBasis(new Plane(new Vector3D(planenormal), 0), new Vector3D(rightvector))
```
Comment (platypii) [inline packages/modeling/src/maths/OrthoNormalBasis.js:109]: Why is the return statement commented out? Isn't this return needed?

Comment (z3dev) [reply packages/modeling/src/maths/OrthoNormalBasis.js:109]: Actually, this function is dead code, using unknown types, and untested. I'd prefer just to remove the unused code. Thoughts?

Comment (z3dev) [reply packages/modeling/src/maths/OrthoNormalBasis.js]: I removed the unused untested code. This is GIT so the code can be recovered anytime.

**Review by platypii — APPROVED** [review]
There's nothing better than removing code

**Pull request merged**